recipe/meta.yaml

```
@@ -38,8 +86,21 @@ test:
     - fastapi.security
   requires:
     - pip
+    - pyjwt 2.9.0
+    - python-multipart
+    - pydantic-settings
+    - pytest >=7.1.3,<9.0.0
+    - inline-snapshot >=0.21.1
+    - flask >=1.1.2,<4.0.0
+    - dirty-equals
+    - sqlalchemy
+    - sqlmodel
+    - orjson
+    - httpx
+    - trio
   commands:
     - pip check
```

https://github.com/fastapi/fastapi/blob/0.121.1/pyproject.toml#L128
```suggestion
    - pip check
    - fastapi --help
```